lib/graphql/query/arguments.rb

```diff
@@ -115,7 +133,7 @@ def wrap_value(value, arg_defn_type)
             wrap_value(value, arg_defn_type.of_type)
           when GraphQL::InputObjectType
             if value.is_a?(Hash)
-              arg_defn_type.arguments_class.new(value)
+              arg_defn_type.arguments_class.new(value, Hash.new(false))
```

Comment:
I think this is a sensible value to pass back to the initializer if an argument default happens to be a Hash.

Reply:
Yeah, it makes sense to me, too. But it seems like this code is a bug waiting to happen, huh? I mean, since default values are applied elsewhere, I wonder if there's some case where a default value isn't properly applied. :S 

Anyhow, that doesn't have to do with these changes!